utils/Sounds.ts

```diff
@@ -0,0 +1,49 @@
+import { Audio } from "expo-av";
+
+enum Sounds {
+  NEXT = require("app/assets/sounds/nextbeep.mp3"),
+  FIRST = require("app/assets/sounds/firstbeep.mp3"),
+}
+
+const firstSound = new Audio.Sound();
+const nextSound = new Audio.Sound();
+
+async function loadSound() {
+  // Loading Sound
+  try {
+    await firstSound.loadAsync(Sounds.FIRST);
+    await nextSound.loadAsync(Sounds.NEXT);
+  } catch (error) {
+    console.log(error);
+  }
+}
+
+loadSound();
```

Comment:
It's just my preference, but I wanna call this function right before `export default { playFirstSound, playNextSound };` so that we can make it clear that declarations are on top of the file and logics are on the lower side.